app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/ui/tracker_activity/DeviceShieldTrackerActivity.kt

```diff
@@ -101,7 +101,7 @@ class DeviceShieldTrackerActivity :
 
     private val feedConfig = DeviceShieldActivityFeedFragment.ActivityFeedConfig(
         maxRows = MIN_ROWS_FOR_ALL_ACTIVITY,
-        timeWindow = 5,
+        timeWindow = 7,
```

Comment:
Changed from 5 to 7 as agreed in comments on https://app.asana.com/0/0/1204022302262178/f